mass/core/channel_group.py

```diff
@@ -631,9 +631,9 @@ def calc_external_trigger_timing(self, forceNew=False):
                    ("subframes_from_nearest_external_trigger" in ds.hdf5_group) and (not forceNew):
                     continue
 
-                subframes_after_last, subframes_until_next = \
-                    mass.core.analysis_algorithms.nearest_arrivals(ds.p_subframecount[:],
-                                                                   external_trigger_subframe_count)
+                NA = mass.core.analysis_algorithms.python_nearest_arrivals
+                subframes_after_last, subframes_until_next = NA(ds.p_subframecount[:],
+                                                                external_trigger_subframe_count)
                 nearest = np.fmin(subframes_after_last, subframes_until_next)
```

Comment:
`nearest` should be a signed quantity to indicate if the nearest trigger was before of after, by using `fmin` you throw away information

Reply:
I think someone in the past argued that IF you care about the sign of nearest, you can get it from the after_last/until_next. So the existence of nearest is only interesting when you don't care about sign. Fair?

Reply:
By that argument the existing of `nearest` is never interesting because you can also get the absolute value from `after_last`/`until_next`. I think nearest is never used, so I'd be fine with removing nearest, but I feel strongly that signed is better than unsigned. I'd rather `abs` one number to get unsigned than refer to 2 numbers to get the sign.